SharpPcap/PosixTimeval.cs

```diff
@@ -241,6 +241,18 @@ public PosixTimeval(ulong Seconds, ulong MicroSeconds)
             this.MicroSeconds = MicroSeconds;
         }
 
+        /// <summary>
+        /// Construct a PosixTimeval using the current UTC time
```

Comment:
Comment looks like it needs to be updated from the copy of the other constructor